docs/ai/specs/agent-task-scheduling.md

```
@@ -0,0 +1,229 @@
+  "lease_expires_at": "...",  // reverts to ready if not completed in time
+  "attempts": 1,
+
+  // terminal
+  "completed_at": "...",
+  "result": "summarized 3 sessions"
+}
+```
+
+### Lifecycle
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)